tests/core/services/config.py

```diff
@@ -1,5 +1,3 @@
 from __future__ import annotations
 
 parallel = False
-install_timelord = False
-checkout_blocks_and_plots = False
```

Comment:
these are the default values, right? so removing them shouldn't have an effect iiuc.